### PR TITLE
[ENH] update CI badges to accomodate several travis and github actions

### LINKED ISSUE #66: A few badges broken
Hey BIDS apps team! It looks like a column of badges are 404-ing on https://bids-apps.neuroimaging.io/apps/:

image

I suspect a circle workflow was removed, changed, or something along those lines, it looks like the Circle URL that is linked to is 404 along with the badge. https://circleci.com/gh/BIDS-Apps/aa.svg?style=shield

Just wanted to let you know!

### LINKED ISSUE #70: allow apps to run their CI on any CI provider
### Your idea

Currently we can only show CI badges on our app page if the app runs its CI on circle-CI.

This is too restrictive.

We should probably allow apps to run their CI anywhere and adapt how those badges are grabbed for each case.

We could cover:
- travis (see bidsonym #66)
- github action (fitlins could be a candidate)

### PR DESCRIPTION
**Pull request by Remi-Gau**
closes #70 
closes #66

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,3 +2,5 @@ source 'https://rubygems.org'
 
 gem "github-pages", group: :jekyll_plugins
 gem "jekyll-paginate"
+
+gem "webrick", "~> 1.7"
```
Comment (Remi-Gau) [inline Gemfile:6]: adding this in the gemfile because it is very often required to install it anyway.

File: apps.md [hunk 2]
```diff
@@ -9,42 +9,59 @@ layout: default
...
+          {% if app.ci == "travis" %}
+              <a href="https://app.travis-ci.com/{{ app.gh }}">
+                <img src="https://app.travis-ci.com/{{ app.gh }}.svg?branch={{ branch }}" />
+              </a>
+          {% elsif app.ci == "gh" %}
+              <a href="https://github.com/{{ app.gh }}/actions/workflows/{{ app.workflow }}.yml/">
+                <img src="https://github.com/{{ app.gh }}/actions/workflows/{{ app.workflow }}.yml/badge.svg?branch={{ branch }}" />
+              </a>
+          {% elsif app.ci == "none" %}
+            <img src="https://img.shields.io/badge/CI-unavailable-lightgrey" />
```
Comment (Remi-Gau) [inline apps.md:43]: if not CI is set up that should make it more obvious

**Comment by Remi-Gau** [comment]
This leads to some badge heterogeneity but I can personally live with this.


Screenshot from 2022-12-10 13-03-25

**Review by sappelhoff — APPROVED** [review]
awesome work @Remi-Gau, thanks!

**Pull request merged**